Mesh4all/m4a-firmware
# boards: merge repo. boards as folder

### Contribution description
First step in the groupping repositories task

### Testing procedure
make should work with m4a boards:

```sh
make BOARD=m4a-mb
```
or
```sh
make BOARD=m4a-24g
```
should build the firmware

### Issues/PRs references
#69
